maquetado/scss/navbar.scss

```diff
@@ -0,0 +1,35 @@
+@import 'variables';
+
+.nav {
+  height: 0;
+  left:0;
+  margin: auto;
+  position: fixed;
+  top: 0;
+  z-index: 9999;
+}
+
+.navbar {
+  background-color: white;
+  border-bottom: 1px lightgray solid;
+  box-shadow: 0 2px lightgray;
+  display: flex;
+  height: $nav-height;
+  justify-content: space-between;
+  position: fixed;
+  width: 100%;
+}
+
+.logo {
+  padding: 20px;
```

Comment:
The padding should be in the `.navbar` not here

Reply:
Very good approach, thx